src/index.html

```diff
@@ -4,9 +4,295 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <!-- <link rel="stylesheet" href="./styles/main.scss"> -->
+    <link rel="stylesheet" href="./styles/styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet"> -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="Moyo_Logo"
+        class="Moyo_image"
+      >
+      <nav class="navbar">
+        <a href="index.html" class="navbar__item is-active">APPLE</a>
+        <a href="index.html" class="navbar__item">SAMSUNG</a>
+        <a href="index.html" class="navbar__item">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+        <a
+          href="index.html"
+          data-qa="nav-hover"
+          class="navbar__item"
+        >
+          НОУТБУКИ И КОМПЬЮТЕРЫ
+        </a>
+        <a href="index.html" class="navbar__item">ГАДЖЕТЫ</a>
+        <a href="index.html" class="navbar__item">ПЛАНШЕТЫ</a>
+        <a href="index.html" class="navbar__item">ФОТО</a>
+        <a href="index.html" class="navbar__item">ВИДЕО</a>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="items">
+        <div class="item" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            class="item__photo"
+            alt="Imac_photo"
+          >
+          <div class="item__desc">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </div>
+          <div class="item__code">Код товара: 195434
+          </div>
+          <div class="item-ratings">
+            <div class="item-stars">
+              <div class="item-stars__star-active"></div>
+              <div class="item-stars__star-active"></div>
+              <div class="item-stars__star-active"></div>
+              <div class="item-stars__star-active"></div>
+              <div class="item-stars__star-not-active"></div>
+            </div>
+            <div class="item-feedback">Отзывов: 5</div>
+          </div>
+          <div class="price-container">
+            <div class="price-container__tag">Цена:</div>
+            <div class="price-container__value">69 999 грн</div>
+          </div>
+          <a
+            class="link-button"
+            href="index.html"
+            data-qa="card-hover"
+          >
+          КУПИТЬ
```

Comment:
```suggestion
            КУПИТЬ
```
Тут тексто тоже обычный, а через css - uppercase